configurations/analysis_soepy_config.py

```diff
@@ -11,5 +11,5 @@
 # We rely on relative paths throughout the package.
 PACKAGE_DIR = Path(os.path.abspath(__file__)).parents[1]
```

Comment:
Please rename to `PROJECT_DIR` and use our environment variable.

Reply:
Can you elaborate on this and above again: I should rename PACKAGE_DIR to PROJECT_DIR and then change what in .envrc?